Source_Code/TM_UnitTests/TeamMentor.UnitTests/CoreLib/Schemas/Test_Schemas_Validation.cs

```diff
@@ -75,15 +75,17 @@ public void Validation_NewUser_LargeDataInFields()
                              "mappings : {0}".format(result.Value.toString()));
         }
         
-        [Test]
+        [Test,Ignore]

```

Comment:
ok, but how do you know that the new method doesn't have performance implications? Maybe the  ```new System.Net.Mail.MailAddress(tmUser.Email);``` will do an regex internally :)

This is the kind of test that should be left in place, specially when the backend method changes (since we want to make sure that the new method does not have the same probs we had before)